spec/features/admin/admin_deletes_restaurant_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+feature 'admin deletes restaurant', %{
+  As an administrator of Date Bites
+  I want to delete a restaurant
+  So that I can remove closed restaurants from the site.
+} do
+
+  # Acceptance Criteria:
+  # [X] I must be logged in as an admin; if I am not, when I visit
+  # /admin/restaurants - I receive an error and then redirected to the home page
+  # [X] I can delete a restaurant by visiting admin/restaurants and clicking on
+  # a delete link next to the entry associated to the restaurant
+  # [X] I receive a confirmation when a restaurant has been deleted
+  # [X] After deleting a restaurant, I am redirected to the admin/restaurants
+  # page
+
+  let!(:admin) { FactoryGirl.create(:user, role: 'admin') }
+  let!(:restaurant) { FactoryGirl.create(:restaurant) }
+
+  scenario 'admin deletes restaurants' do
+    sign_in_as(admin)
+    visit '/admin/restaurants'
+
+    page.click_link("Delete", href: "/admin/restaurants/#{restaurant.id}")
```

Comment:
There should only be one restaurant on the page. You should be able to change this to `click_on "Delete"`

Another note, webcrawlers, such as Google, will visit every link on a page. If you have a link to delete a record on a page, a webcrawler may delete your data! Use a button for anything that is additive or subtractive.